src/frontend/web_application/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# This file creates a container that runs a {package} caliopen frontend
+# Important:
+# Author: Caliopen
+# Date: 2016-01-05
+
+FROM node:6
+MAINTAINER Caliopen
+
+ADD . /srv/caliopen/frontend/
+WORKDIR /srv/caliopen/frontend/
+RUN yarn install
+RUN yarn run release
```

Comment:
As discussed with gdchamal, let's go with Docker containers that are 'production' ready.  
If a developer needs to put his hands in a specific code part, it's better for him to not run the container but run the code barely on its computer. For example, a frontend dev. could launch `redis cassandra elasticsearch`, then `api`, but not `frontend`. And build&run the frontend directly.